walkthroughs/fargate/README.md

```diff
@@ -0,0 +1,231 @@
+# Using AWS App Mesh with Fargate
+
+## Overview
+
+I gave a walkthrough in my [previous article] on how to deploy a simple microservice application to ECS and configure [AWS App Mesh] to provide traffic control and observability for it. In this article, we are going to start to explore what it means when we say that App Mesh is a service mesh that lets you control and monitor services spanning multiple AWS compute environments. We'll start with using Fargate as an ECS launch type to deploy a specific version of our `colorteller` service before we move on and explore distributing traffic across other environments, such as EC2 and EKS.
```

Comment:
Can you link the bracketed items to appropriate destinations? These will be broken links otherwise.

Reply:
These are Markdown implicit links; the links are fully specified at the end of the doc (keeps the doc uncluttered). It's part of the standard and supported by GFM.